pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/AbstractJavaNode.java

```diff
@@ -273,7 +273,7 @@ void replaceChildAt(int idx, JavaNode newChild) {
     }
 
     @Override
-    public final String getXPathNodeName() {
+    public String getXPathNodeName() {
```

Comment:
See comment above: It might be surprising if a node has a different name in XPath than in Java...